shared/src/commonMain/kotlin/org/jetbrains/kotlinconf/screens/SessionScreen.kt

```diff
@@ -126,62 +128,80 @@ fun SessionScreen(
             transitionSpec = { FadingAnimationSpec }
         ) { hasState ->
             if (hasState && session != null) {
-                Column(
-                    modifier = Modifier
-                        .padding(horizontal = 12.dp)
-                        .padding(bottomInsetPadding())
-                        .verticalScroll(rememberScrollState())
-                ) {
-                    PageTitle(
-                        time = session.fullTimeline,
-                        title = session.title,
-                        lightning = session.isLightning,
-                        tags = session.tags,
-                        bookmarked = session.isFavorite,
-                        onBookmark = { viewModel.toggleFavorite(it) },
-                        modifier = Modifier.padding(vertical = 24.dp),
-                    )
+                val listState = rememberLazyListState()
 
-                    if (session.state != SessionState.Upcoming) {
-                        FeedbackPanel(
-                            onFeedback = { emotion ->
-                                viewModel.submitFeedback(emotion)
-                            },
-                            onFeedbackWithComment = { emotion, comment ->
-                                viewModel.submitFeedbackWithComment(emotion, comment)
-                            },
-                            userSignedIn = userSignedIn,
-                            initialEmotion = session.vote?.toEmotion(),
-                            feedbackQuestion = stringResource(
-                                if (session.tags.contains("Workshop")) UiRes.string.talk_card_how_was_the_workshop
-                                else UiRes.string.talk_card_how_was_the_talk
-                            ),
-                            modifier = Modifier.padding(bottom = 20.dp),
-                        )
+                LaunchedEffect(openedForFeedback) {
+                    if (openedForFeedback) {
+                        listState.animateScrollToItem(1)
                     }
+                }
+
+                ScrollToTopHandler(listState)
 
-                    speakers.forEach { speaker ->
-                        SpeakerCard(
-                            name = speaker.name,
-                            title = speaker.position,
-                            photoUrl = speaker.photoUrl,
-                            modifier = Modifier.padding(vertical = 12.dp).fillMaxWidth(),
-                            onClick = { onSpeaker(speaker.id) }
+                LazyColumn(
```

Comment:
Swapped this to a LazyColumn so we can scroll precisely to its second item for the feedback block